functions/package.json

```diff
@@ -14,10 +14,14 @@
   },
   "main": "lib/index.js",
   "dependencies": {
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
```

Comment:
At this time, These package is not used.
But, in the near future, these package should be used.
So, this is OK.